src/main/java/by/matsukiryna/arraytask/service/impl/ArraySortServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package by.matsukiryna.arraytask.service.impl;
+
+import by.matsukiryna.arraytask.entity.ArrayEntity;
+import by.matsukiryna.arraytask.exception.ArrayException;
+import by.matsukiryna.arraytask.service.ArraySortService;
+
+public class ArraySortServiceImpl implements ArraySortService {
+    @Override
+    public void bubbleSort(ArrayEntity arrayEntity) throws ArrayException {
+        int[] tmpArray = arrayEntity.getArrayEntity();
+        int tmp;
+        for (int i = 1; i < tmpArray.length; i++) {
+            for (int j = tmpArray.length - 1; j >= i; j--) {
+                if (tmpArray[j - 1] > tmpArray[j]) {
+                    tmp = tmpArray[j - 1];
+                    tmpArray[j - 1] = tmpArray[j];
+                    tmpArray[j] = tmp;
+                }
+            }
+        }
+        arrayEntity.setArrayEntity(tmpArray);
+    }
+
+    @Override
+    public void insertionSort(ArrayEntity arrayEntity) {
```

Comment:
sorting direction in all method names can be more descriptive

Reply:
the mentor asks to name methods close to the standard library